code/modules/mob/mob.dm

```diff
@@ -628,9 +628,30 @@ GLOBAL_LIST_INIT(slot_equipment_priority, list( \
 		return 1
 
 	face_atom(A)
-	var/list/result = A.examine(src)
+	var/list/result
+	if(client)
```

Comment:
make this an early return

```dm
if(!client)
	result = A.examine(src) // if a tree is examined but no client is there to see it, did the tree ever really exist?
	to_chat(src, chat_box_examine(result.Join("\n"))
	return
```

Reply:
tell me if did both reviews properly right :^)